src/features/rewards/mediaBox/index.tsx

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import {
+  StyledMediaBox,
+  StyledMediaText,
+  StyledMediaTimestamp,
+  StyledMediaIcon
+} from './style'
+
+import {
+  TwitterColorIcon,
+  RedditColorIcon
+} from '../../../components/icons'
+
+export interface Props {
+  mediaType: 'twitter' | 'reddit'
+  mediaText: string
+  mediaTimestamp: number
```

Comment:
why do we need `mediaTimestamp` and `mediaTimetext`? Wouldn't time displayed in the box be the same?

Reply:
not sure why twitter would have different format then reddit

Reply:
we should try to have box as general as possible so when we add github we would just add timestamp in and everything would just work

Reply:
Addressed via DM